internal/conf/config.go

```diff
@@ -54,10 +54,10 @@ type OperatorConfig struct {
 	EnvoyProxyImageTag string `envconfig:"ENVOY_PROXY_IMAGE_TAG" default:"v1.21-latest"`
 
 	// WorkerOSBuildComposerVersion is the release tag of OSBuild-Composer for the osbuild-worker RPM
-	WorkerOSBuildComposerVersion string `envconfig:"WORKER_OSBUILD_COMPOSER_VERSION" default:"v56"`
+	WorkerOSBuildComposerVersion string `envconfig:"WORKER_OSBUILD_COMPOSER_VERSION" default:"v59"`
```

Comment:
Why the version is different from the worker version?

Reply:
OSBuild (from which the `osbuild` RPM is release) and OSBuild-Composer (from which the `osbuild-composer-worker` RPM is released) are managed in different repositories, have different release cycles and hence are not on the same version number